js/dashboard.js

```diff
@@ -199,6 +199,110 @@ $(function() {
         }
     });
 
+    $('#leafwetness-button').click(function() {
+				$.ajax({
+            url: 'http://localhost:8000/env_data?vineyard_id=0&env_variable=leafwetness',
+            type: "GET"
+        }).done(function(json) {
+            if (json.env_data) {
+                window.location.href = "http://localhost:8000/env_data?vineyard_id=0&env_variable=leafwetness";
+            } else {
+                // go nowhere
```

Comment:
These "go nowhere" statements seem unnecessary. shouldn't we just do nothing at all? or is there a reason we're specifying a window location to no where?